sdk/data_cosmos/src/authorization_policy.rs

```diff
@@ -57,11 +57,11 @@ impl Policy for AuthorizationPolicy {
 
         let time_nonce = TimeNonce::new();
 
-        let uri_path = request.path_and_query();
+        let uri_path = &request.path_and_query()[1..];
```

Comment:
This matches what the code was, prior to this change:
https://github.com/Azure/azure-sdk-for-rust/pull/811/files#diff-1d8d32914ddb120499d9acbf8fb856f3757b01bf2d612c4eef44e5ff9590dcbaL57

I think in #811, there was a switch from `http::Uri` to `url::Url` and `path_and_query` existed in the former.

https://docs.rs/http/latest/http/uri/struct.Uri.html#method.path_and_query
https://docs.rs/url/latest/url/struct.Url.html#method.path